.changeset/wise-loops-tell.md

```diff
@@ -0,0 +1,19 @@
+---
+"@fluidframework/tree": minor
+"__section": tree
+---
+Allows users of alpha schema APIs to add metadata to AllowedTypes
+
+This change allows metadata to be added to AllowedTypes as well as individual types in a set of AllowedTypes.
```

Comment:
references to type names should be code formatted, and the first reference should be linked.

```suggestion
This change allows metadata to be added to [`AllowedTypes`](https://fluidframework.com/docs/api/fluid-framework/allowedtypes-typealias) as well as individual types in a set of `AllowedTypes`.
```